test/helpers/add-agreement-to-closure-db.js

```diff
@@ -0,0 +1,13 @@
+const db = require('../../app/data')
+
+const addAgreementToClosureDB = async (schemeId, frn, agreementNumber) => {
+  const closedId = 1
+  const date = new Date()
+  const closureDate = date.getFullYear + '-' + date.getMonth() + '-' + date.getDate()
+  const savedAgreementClosure = await db.frnAgreementClosed.create({ closedId, schemeId, frn, agreementNumber, closureDate })
```

Comment:
Might be best to create a mock object in the `mocks` folder.  Then in the `beforeEach` of your test just call `await db.frnAgreementClosed.create(whateverYouReferToTheMockAs)` to save needing a helper method?

Reply:
created helper constants and simplified creation process